src/ui/components/Account/Account.tsx

```diff
@@ -10,18 +10,19 @@ import type {Account as AccountType} from 'src/api/hooks/useAccount';
 type Props = {
   account: AccountType;
   name?: string;
+  onPress?: () => void;
 };
 
-export function Account({account, name}: Props) {
+export function Account({account, name, onPress}: Props) {
```

Comment:
This component should not have `onPress`, if needed the parent component should handle the `onPress`